hw/dv/tools/dvsim/dsim.hjson

```diff
@@ -23,6 +23,8 @@
                "-c-opts -I{DSIM_HOME}/include",
                "-timescale 1ns/1ps",
                "-f {sv_flist}",
+               // List multiple tops for the simulation. Prepend each top level with `-top`.
+               '''{eval_cmd} tops=(`echo {sim_tops}`); echo "${tops[@]/#/-top }"''',
```

Comment:
While we discuss the escaping in the previous commit, you could do this with sed (and no escaping required):
```
echo {sim_tops} | sed -E 's/(\w+)/-top \1/g'
```
Maybe split out the previous commit into a separate PR, and we can figure out the best approach for that separately from this change?